nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -769,14 +769,16 @@ where
         let blobs_ids = get_sampled_blobs(sampling_relay);
         match futures::join!(cl_txs, da_certs, blobs_ids) {
             (Ok(cl_txs), Ok(da_blobs_info), Ok(blobs_ids)) => {
-                let Ok(block) = BlockBuilder::new(tx_selector, blob_selector)
-                    .with_cryptarchia_builder(Builder::new(parent, slot, proof))
-                    .with_transactions(cl_txs)
-                    .with_blobs_info(
-                        da_blobs_info.filter(move |info| blobs_ids.contains(&info.blob_id())),
-                    )
-                    .build()
-                else {
+                let Ok(block) = BlockBuilder::new(
+                    tx_selector,
+                    blob_selector,
+                    Builder::new(parent, slot, proof),
+                )
+                .with_transactions(cl_txs)
+                .with_blobs_info(
+                    da_blobs_info.filter(move |info| blobs_ids.contains(&info.blob_id())),
+                )
+                .build() else {
                     panic!("Proposal block should always succeed to be built")
                 };
```

Comment:
Isn't this pattern equiv. to `.expect("Proposal block should always succeed to be built")`?

Reply:
yes, but I didn't change this code, just moved it